<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::SelectBeneficiariesDialog</class>
 <widget class="QDialog" name="tt3::gui::SelectBeneficiariesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select beneficiaries</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tt3-gui.qrc">
    <normaloff>:/tt3-gui/Resources/Images/Objects/BeneficiaryLarge.png</normaloff>:/tt3-gui/Resources/Images/Objects/BeneficiaryLarge.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTreeWidget" name="beneficiariesTreeWidget">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="filterWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="filterLabel">
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="filterLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QListWidget" name="beneficiariesListWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>64</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="selectedBeneficiariesLabel">
     <property name="text">
      <string>Selected beneficiaries:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>beneficiariesTreeWidget</tabstop>
  <tabstop>beneficiariesListWidget</tabstop>
  <tabstop>filterLineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tt3::gui::SelectBeneficiariesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tt3::gui::SelectBeneficiariesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tt3::gui::SelectBeneficiariesDialog</receiver>
   <slot>_filterLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beneficiariesTreeWidget</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>tt3::gui::SelectBeneficiariesDialog</receiver>
   <slot>_beneficiariesTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beneficiariesListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>tt3::gui::SelectBeneficiariesDialog</receiver>
   <slot>_beneficiariesListWidgetCurrentRowChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_filterLineEditTextChanged(QString)</slot>
  <slot>_beneficiariesTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
  <slot>_beneficiariesListWidgetCurrentRowChanged(int)</slot>
 </slots>
</ui>
